tasks/43_brackets-indexes/brackets-indexes.js

```diff
@@ -0,0 +1,11 @@
+export function findCloseIndex(str, bracketInd) {
+  if (str[bracketInd] === '(') {
```

Comment:
Indentation can be reduced with early return:
```suggestion
  if (str[bracketInd] !== '(') {
  return -1
  }
  // ... code...
```